### DISCUSSION TITLE
onmicrosoft.com email delay

### DISCUSSION THREAD

**Discussion by calvez** [modoboa/modoboa#2657] [Help]
On my modoboba server we experience serious delays when the sender comes from office 365 or exchange server.
The sender is usually ....onmicrosoft.com and mails are delayed 3-4 or even 8 hours.
The sender usually didn't get any response and the mails are delivered but this delay is unacceptable for my client.
What can I do?
Any suggestion?

**Comment by MrGeneration** [comment]
Disabling grey listing is imho your only option.

**Comment by calvez** [comment]
Thank you I did it and will let you know the result.

**Comment by tonioo** [comment]
Or you can whitelist microsoft IPs using postwhite (already installed).
